.gitignore

```diff
@@ -87,6 +87,7 @@ jarjar-rules.txt
 /fastlane/Preview.html
 /fastlane/Deployment
 /Preview.html
+gc_keys.json
```

Comment:
Include the directory in the ignore path, too.

Reply:
Do you mean the directory where credentials are stored? Match installs them directly into your system, so they won't be in the repo directory. I've added `gc_keys.json` only in case - this is the default approach when you don't authenticate via `gcloud auth application-default login` command and you only run `fastlane match` - it will you to generate a `gc_keys.json` file.

Reply:
I just meant to specify a full path in the .gitignore file (like we do with `/Preview.html` above, for example).